distribution/docker/ddf/pom.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version. 
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>ddf.distribution</groupId>
+        <artifactId>docker</artifactId>
+        <version>2.14.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>ddf</artifactId>
+    <groupId>ddf.distribution.docker</groupId>
+    <packaging>pom</packaging>
+    <name>DDF ::  Distribution :: Docker :: DDF</name>
+
+    <properties>
+        <docker.ddf.base.version>2.14-alpine</docker.ddf.base.version>
```

Comment:
❓Is the `2.14` alpine version related to the DDF version? If so, should we use something like the `build-helper-maven-plugin` to parse the project version (see https://stackoverflow.com/a/18234772) and use those properties instead of hard-coding `2.14`? It'll be really easy to forget to update this otherwise.

Reply:
❓Should we move this to the top-level pom or is this too specific to this module?

Reply:
This will eventually be unnecessary so I'm thinking of just leaving it like this for now

Reply:
❓ Does this mean the Solr docker definition files will become part of the actual DDF distribution?